examples/resnet-concurrent.cpp

```diff
@@ -196,9 +196,12 @@ int main(int argc, char **argv) {
                                   ImageChannelOrder::BGR, ImageLayout::NCHW,
                                   /* useImagenetNormalization */ true);
 
+    Tensor batch(inputType);
+    batch.getHandle<float>().insertSlice(image, 0);
```

Comment:
Can we instead get an unowned tensor of `image` with the correct dimensions? Using insertSlice will require copying the image data.